src/puppy/platforms/macos/platform.nim

```diff
@@ -18,6 +18,9 @@ proc fetch*(req: Request): Response {.raises: [PuppyError].} =
     if req.body.len > 0:
       request.setHTTPBody(NSData.dataWithBytes(req.body[0].addr, req.body.len))
 
+    if req.insecure:
+      request.setAllowsAnyHTTPSCertificate()
```

Comment:
This probably needs to be:
```nim
request.setAllowsAnyHTTPSCertificate(true, @url.host)
```

Reply:
I'll also try this

Reply:
This can be deleted too, we'll not be doing anything in macos/platform.nim